pkg/cmd/cobra/cobra.go

```diff
@@ -280,7 +285,7 @@ func GetTraceeRunner(c *cobra.Command, version string) (cmd.Runner, error) {
 
 	p, err := printer.NewBroadcast(
 		output.PrinterConfigs,
-		cmd.GetContainerMode(containerFilterEnabled(), cfg.NoContainersEnrich),
+		cmd.GetContainerMode(containerFilterEnabled(), cfg.EnrichmentEnabled),
```

Comment:
Shouldn't this bool be reverted now with this change?

Reply:
It was really hard to head using it as a true for no internally, so I've changed internally to use a normal true/false to enable/disable.